parsecmdline_pk.lua

```diff
@@ -32,13 +32,15 @@ local function getopts(opt_meta, arg, usage)
     local apos = 1 + (opt_meta[0] or 0)
 
     local skipnext = false
+    local proc = true
     for i=1,#arg do
         if (skipnext) then
             skipnext = false
             goto next
         end
 
-        if (arg[i]:sub(1,1)=="-") then
+        if arg[i] == "--" then proc = false end
```

Comment:
Please note the coding convention: the condition to "if" is surrounded by parens, and the body goes to a separate line. Yes, I know the former is not according to Lua conventions, but I find it easier to read. Same for the latter.